website/docs/docs/dbt-cloud/cloud-overview.md

```diff
@@ -4,7 +4,7 @@ id: "cloud-overview"
 ---
 
 
-[dbt Cloud](https://www.getdbt.com/product/) is a hosted service that helps data analysts and engineers productionize dbt deployments. It comes equipped with turnkey support for scheduling jobs, CI/CD, serving documentation, monitoring & alerting, and an Integrated Developer Environment (IDE).
+[dbt Cloud](https://www.getdbt.com/product/) is a hosted service that helps data analysts and engineers productionize dbt deployments. It comes equipped with turnkey support for scheduling jobs, CI/CD, serving documentation, monitoring & alerting, and an integrated developer environment (IDE).
```

Comment:
```suggestion
[dbt Cloud](https://www.getdbt.com/product/) is a hosted service that helps data analysts and engineers productionize dbt deployments. It comes equipped with turnkey support for scheduling jobs, CI/CD, serving documentation, monitoring & alerting, and an integrated development environment (IDE).
```

Reply:
if we want to use `IDE` in the same way the industry is using it, it should be [this (google search)](https://www.google.com/search?q=ide+stands+for&ei=jbTiYsG4MfzNkPIP-r2AwAQ&ved=0ahUKEwiBs6P8-5v5AhX8JkQIHfoeAEgQ4dUDCA4&uact=5&oq=ide+stands+for&gs_lcp=Cgdnd3Mtd2l6EAMyCggAELEDEIMBEEMyBQgAEJECMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQ6BwgAEEcQsAM6BwgAELADEEM6CwgAELEDEIMBEJECOgQIABBDOgsILhCABBDHARCvAToHCC4QgAQQCkoFCDwSATJKBAhBGABKBAhGGABQqgJYtg1giw9oAnAAeACAAW6IAfoGkgEDOS4xmAEAoAEByAEKwAEB&sclient=gws-wiz) unless we are intentionally wanting to dbt-ize/overload the term

Reply:
@bossnunta do we want to keep this "developer?" I see Drew's [blog post](https://www.getdbt.com/blog/announcing-the-dbt-ide/) uses "developer" so I am inclined to keep it as that but it's your call!

Reply:
@nghi-ly Let's leave this as-is and come back to it.

Reply:
oh shoot -- sorry I didn't review this before -- IDE should stand for integrated **development** environment, not developer. I think the original note was a typo.